themes/stanford_basic/src/scss/components/hero/_hero.scss

```diff
@@ -1,9 +1,56 @@
 @charset "UTF-8";
 
 // Banner aspect ratio
-
 .su-hero__media {
   @include grid-media-max('sm') {
     aspect-ratio: 30/11;
   }
 }
+
+// Page Title Banner styles
+.ptype-stanford-page-title-banner {
+  .su-hero {
```

Comment:
do you need this selector? It would be nice to keep the number of nested levels as low as possible.

Reply:
Not needed - removed selector.